src/lsptoolshost/server/roslynLanguageServer.ts

```diff
@@ -255,6 +255,11 @@ export class RoslynLanguageServer {
         channel: vscode.LogOutputChannel,
         traceChannel: vscode.OutputChannel
     ): Promise<RoslynLanguageServer> {
+        const devKit = getCSharpDevKit();
+        if (devKit) {
+            await devKit.activate();
```

Comment:
Allowing the exception to propagate will still produce an error toast in VS Code. But I don't know... maybe it'll be smaller?

Reply:
I don't believe this will show a toast - this is in a promise that isn't awaited in activation and not part of a callback from vscode .  We write an error, but I haven't managed to get a toast to display in any of the editors - I only get a devkit toast.